volumes/miovision/sql/views/create-view-open_issues.sql

```diff
@@ -26,18 +44,19 @@ JOIN miovision_api.intersections AS i USING (intersection_uid)
 --find last week volume
 LEFT JOIN miovision_api.volumes AS v
     ON ar.intersection_uid = v.intersection_uid
+    --volume within the last 7 days and after AR started
+    AND v.datetime_bin >= ar.range_start
```

Comment:
so for example if we run this today, and the range was add yesterday, 
the sum volume for `last_week_volume` would be summing up only 1 day of data

Reply:
Yes, because we wouldn't want to know the volume from before `range_start`, which would either be zero, or non-zero from a time before the issue started. I could rename it/change it to average daily volume if you think that's better?

Reply:
yess, to reduce confusion if that ever happens

Reply:
Decided to explain it with a column comment and also in the `check_open_anomalous_ranges` airflow alert.